avocado/core/nrunner.py

```diff
@@ -185,6 +186,8 @@ class ExecRunner(BaseRunner):
     """
     def run(self):
         env = self.runnable.kwargs or None
+        if env and 'PATH' not in env:
```

Comment:
I understand your goal here, and yes, there's a use case for this.

But, one of the things I wonder about, and that I would like very much to be able to achieve, is to have more reproducible results in an Avocado job execution, and the test execution is key. If we add such dynamic inputs to the "runnable" execution, we lose that. For instance, if a runnable recipe for  `exec` or `tap` is making use of this dynamic `PATH`, then it would perform differently on different systems, even though the input (described on the recipe) is the same.

I hope my comment is somewhat clear, but we can expand on that topic.

Reply:
@clebergnu not sure if I got your point. But yes, when I submitted this, I was not 100% sure of this. My only intention was to avoid users need to pass PATH everytime.